src/author/author.service.ts

```diff
@@ -31,32 +34,35 @@ export class AuthorService {
     const { name } = updateDto;
     const found = await this.getAuthorById(id);
     if (found) {
-      console.log(found);
-      return await this.authorRepository.save({
-        id: id,
-        name: name,
-      });
+      found.name = name;
+      return await this.authorRepository.save(found);
```

Comment:
if (!found) {
  throw E;
}

return await this.authorRepository.save({...found, name })

Reply:
done